### DISCUSSION TITLE
clairctl  failed to create baggage

### DISCUSSION THREAD

**Discussion by eldoranstars** [quay/clair#1693] [Q&A]
Hey!
when I run:
`clairctl -D report --host http://***.com ***.com/***/***:latest`
At the end report is successfully created, but first I get an error:
`failed to create baggage error "invalid baggage list-member"`

What does it mean? Can you help me plz?

**Comment by hdonnay** [comment]
What version are you using?

**Comment by eldoranstars** [comment]
i used clairctl from here https://github.com/quay/clair/tree/release-4.4/cmd/clairctl
but now i changed it from here https://github.com/quay/clair/tree/release-4.6/cmd/clairctl
previous error has disappeared

P.S.: i found some interesting things
https://github.com/quay/clair/tree/release-4.4/cmd/clairctl has "clairctl version 0.2.0"
https://github.com/quay/clair/tree/release-4.6/cmd/clairctl has "clairctl version (random source build)"
idk why ...

**Reply by hdonnay** [reply]
Yeah, the issue shouldn't be present in new versions.
